PIL/Image.py

```diff
@@ -1426,6 +1503,7 @@ def alpha_composite(self, im, dest=(0,0), source=(0,0)):
         self.paste(result, box)
 
     def point(self, lut, mode=None):
+        # type: (Union[List, Callable[[Any], Any]], Optional[Text]) -> Image
```

Comment:
For a `Callable`, a fully-typed version is `Callable[[PARAM_LIST], RETURNED]`.

The related PEP (484?) or the mypy docs are pretty good for this :)